src/main/java/Duke/UI.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * UI for Duke.
+ */
+public class UI {
```

Comment:
```suggestion
public class Ui {
```
I think abbreviations are not supposed to be uppercase when they are part of a name?